test_cases/stdlib/check_sqlite3.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import sqlite3
+from contextlib import closing
```

Comment:
The `closing` wrapper is not doing much for the tests in typecheck, since the tests are never actually run. It could even be harmful if the closing annotations are ever changed.

Reply:
This is done (use of `closing()` removed): shawnbrown/typeshed@9a13166